swagger/spec-loader.js

```diff
@@ -62,17 +63,27 @@ function loadSpec(specUrlStr, log, cb) {
       }
     });
   } else {
-    fs.readFile(specUrl.href, 'UTF-8', function (err, body) {
-      if (err) {
-        return cb(err);
-      }
+    if(specUrl.href.match(/.yml/)){
```

Comment:
In our coding style, we add spaces after the `if` statement and also between `) {`:

```js
if (specUrl.href.match(/.yml/)) {
```